tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -21,27 +25,43 @@ def pull_project_peds(session_id: str, projects: set):
     """
```

Comment:
not in the scope of this PR, but this docstring should probably get reformatted to fit the gnomad methods convention. maybe something like:
```
    Use the seqr API to retrieve a list of projects' pedigrees. 

    Writes a single pedigree for all projects ('seqr_pedigrees.txt'), using the retrieved pedigrees. 
    Failing projects are written to 'projects_not_pulled.txt'.

    :param session_id: sessionid cookie used for seqr login
    :param projects:  Set of seqr project GUIDs
```